api-server/server/models/donation.json

```diff
@@ -45,7 +45,6 @@
     },
     "customerId": {
       "type": "string",
-      "required": true,
```

Comment:
Is this change intentional?

Reply:
This, since the customer id is not provided when we get the subscription info from paypal.

Reply:
What would be the alternate from PayPal?

Reply:
for the subscriber the only provide name, email and address.

Reply:
So, what would be the alternate uniquely identifiable information / item that we can query from their API while making calls to them. For instance, if we wanted to get all the transactions for a person. If it's the email, could hat be used instead?

Reply:
we can get transactions of a subscription id but not a specific person.
https://developer.paypal.com/docs/api/subscriptions/v1/#subscriptions_transactions

Reply:
I would re-visit this, and add an equivalent.

Reply:
I added an activation mock hook with its headers and body. There does not seem to be an equivalent. 
I could re add the email as customerId to satisfy this requirement.

Reply:
Yup, adding the email should be fine.